fontforgeexe/startui.c

```diff
@@ -393,12 +393,14 @@ exit( 0 );		/* But the event loop should never return */
 }
 
 static void start_splash_screen(void){
+
     GDrawSetVisible(splashw,true);
+
     GDrawSync(NULL);
     GDrawProcessPendingEvents(NULL);
     GDrawProcessPendingEvents(NULL);
 
-    splasht = GDrawRequestTimer(splashw,7000,1000,NULL);
+    splasht = GDrawRequestTimer(splashw,3200,1000,NULL);
 
```

Comment:
If this were changed back to 7000 I'd have no problems with this, but as it stands I can't accept it.

Reply:
Have addressed in #3795